### PR TITLE
Adds latest data from Veekun

### PR DESCRIPTION
**Pull request by Naramsim**
Thanks to all the folks at Veekun

### CODE REVIEW

**Comment by lmerotta** [comment]
Does this contain the new data and sprites of ultra sun / ultra moon ?

**Comment by Naramsim** [comment]
@lmerotta Yes, you can check it out.

**Comment by SirSkaro** [comment]
This PR has been open for half a year. Is there any news on when this update will go live? I'm using PokeAPI for a Discord bot that has over 370k unique users, and the new Pokemon are popular among my users. Is there anything I can do to help or anyone I can contact? Thank you.

**Comment by Naramsim** [comment]
Hi @SirSkaro, first of all I should update the pull request. I'm sure that Veekun added more data in the past months. Then, even if it will be accepted and merged in `master` we will have to wait for @phalt to update the code on the DO server, reload the DB with the new data and then start serving.

I think that, if @phalt is willing, I can update the PR, you (@SirSkaro) can test it and phalt then will merge it and reload the server.

Until @phalt does not answer us, I will not update the PR.

**Comment by SirSkaro** [comment]
Hi @Naramsim, thank you for the quick reply. If you update your PR, I would be happy to test it. I do hope that Phill merges and reloads, I'm planning on building a website and a suite of social-media bots. 

I just read that PokeAPI is shutting down on a text file in @phalt's github. Is this still true, though DO is sponsoring PokeAPI?

**Comment by contolini** [comment]
Thanks for these CSVs @Naramsim. We used them over at https://github.com/contolini/pokedex/issues/1.

**Comment by tdmalone** [comment]
@Naramsim Is this PR still useful to have merged now? If so I'm happy to review (well, as much as one can review CSVs anyway... 😛)

**Comment by sargunv** [comment]
Ditto could potentially be used to validate something like this. You could diff the output of `ditto clone` of before and after the changes to see what was updated, and then diff the output of `ditto analyze` of before and after to make sure the schema didn't change (`analyze` generates a JSON schema of cloned data).

**Comment by Naramsim** [comment]
Yes guys, you can review it. So we will aligned to Veekun. I took the data directly from their master branch. In future we could add https://github.com/veekun/pokedex as a submodule.

File: data/v2/csv/pokemon_species.csv [hunk 1]
```diff
@@ -792,12 +792,17 @@ id,identifier,generation_id,evolves_from_species_id,evolution_chain_id,color_id,
...
-800,necrozma,7,,421,1,4,,-1,3,0,0,120,0,1,0,800,
+794,buzzwole,7,,415,8,10,,-1,45,0,0,120,0,1,0,794,
+795,pheromosa,7,,416,9,12,,-1,45,0,0,120,0,1,0,795,
+796,xurkitree,7,,417,1,6,,-1,45,0,0,120,0,1,0,796,
+797,celesteela,7,,418,5,12,,-1,45,0,0,120,0,1,0,797,
+798,kartana,7,,419,9,12,,-1,45,0,0,120,0,1,0,798,
+799,guzzlord,7,,420,1,6,,-1,45,0,0,120,0,1,0,799,
+800,necrozma,7,,421,1,4,,-1,255,0,0,120,0,1,0,800,
 801,magearna,7,,422,4,12,,-1,3,0,0,120,0,1,0,801,
 802,marshadow,7,,423,4,12,,-1,3,0,0,120,0,1,0,802,
```
Comment (Dyleee) [inline data/v2/csv/pokemon_species.csv:803]: why did you take off these

**Comment by SirSkaro** [comment]
I have been using Naramsim's fork for about 2 months now with my application. It has received over 225k requests in that time, and there haven't been any complaints about data from the new Pokemon from my users. The majority of data is there.

There are some things left to be desired, such as sprites of the new Pokemon and flavor texts. But PokeAPI in general is missing data or odd keying. For example, in the current release of PokeAPI there are no Pokedex entries (pokemon-species' flavor_texts) for Sun, only for Moon. This fork is missing also Ultra Moon flavor entries. However, this fork has enough new content that I think it should be pushed.

I feel that it should also be a priority to merge this branch (or at least update the database). Ultra Sun and Ultra Moon came out a year ago, and a new games are coming out very soon. Any applications using PokeAPI have been without this data, which is a deal breaker/frustration for many users and developers. A Pokemon application that isn't up-to-date is not ever going to be a user's first choice, so a developer is handicapping themself up front for choosing PokeAPI. If Bulbapedia stopped updating their wiki, it wouldn't take long for it to be surpassed by other Pokemon wikis as the most-visited. I don't mean to sound rude - I know this is a joint project done in people's spare time - but I do feel that this is a point of frustration for many developers and a obstacle to PokeAPI's growth.

tl;dr
bump

**Comment by Naramsim** [comment]
I'll check Veekun released some more data

**Comment by cmmartti** [comment]
@Naramsim In the meantime, however, I see no reason why this can't be merged. We can create another pull request for any new data.\

We should also be more proactive about updating PokéAPI when veekun is updated.

**Comment by Naramsim** [comment]
It can't be merged because we need a reviewer. (Could be anyone)

File: data/v2/csv/pokemon_species.csv (hunk 1 above)
Comment (Naramsim) [reply data/v2/csv/pokemon_species.csv:803]: I copy pasted

**Comment by Naramsim** [comment]
Closing in favor of #388
